app/screens/Course/Course19.tsx

```diff
@@ -20,12 +20,9 @@ export const Course19 = () => {
 
   const currency19 = useAppSelector(state => state.course.course['2019']);
 
-  const data19usd = currency19.exchangeRate[16].saleRateNB;
-  const data19eur = currency19.exchangeRate[22].saleRateNB;
-
   return (
     <View style={styles.container}>
-      <Course data19={data19} data19usd={data19usd} data19eur={data19eur} />
+      <Course DATA_19={DATA_19} currency19={currency19} />
```

Comment:
it should be only one param without information about a year, u don't need to use DATA_19 name like param
1. wrong name. u cant use DATA_19 in param, it should be data19={DATA_19}
2. u dont need to create a param with specific year, just yearData is ok